hathor/transaction/storage/rocksdb_storage.py

```diff
@@ -87,6 +89,36 @@ def _tx_to_bytes(self, tx: 'BaseTransaction') -> bytes:
     def _meta_to_bytes(self, meta: 'TransactionMetadata') -> bytes:
         return json_dumpb(meta.to_json())
 
+    def _import_from_other(self, other_storage: TransactionStorage) -> None:
+        """Internal method, do not run without understanding how to use it.
+
+        Effectively migrates the database to be compatible with the stateful validation that sync-v2 expects.
+        """
+
+        from hathor.transaction.transaction_metadata import ValidationState
+
+        self.log.info('importing from old database')
```

Comment:
Maybe add "(and it might take a while. Be patient!)".

Reply:
Done.